### PR TITLE
[sunder] Add sunder strategy

### PR DESCRIPTION
**Pull request by krotos-arch**
Fixes # .

Changes proposed in this pull request:

Sunder is the customized strategy for Sunder protocol. Any governance token been sundered with dToken can choose this strategy to get it included with the original token with 1:1 voting power.

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hey, Please remove the changes from `yarn.lock`, Everything else looks good to me :)

File: src/strategies/sunder/index.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.balanceOfAdds, 'balanceOfDToken', [
+      options.token,
+      address
+    ])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  console.log(result);
```
Comment (ChaituVR) [inline src/strategies/sunder/index.ts:31]: Hey please remove these unwanted logs, everything else seems good to go
```suggestion
```

File: test/index.spec.ts [hunk 2]
```diff
@@ -85,6 +85,8 @@ describe(`\nTest strategy "${strategy}"`, () => {
   });
 
   it('Returned addresses should be either same case as input addresses or checksum addresses', () => {
+
+
     expect(
```
Comment (ChaituVR) [inline test/index.spec.ts:90]: Please remove these new lines
```suggestion
    expect(
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by krotos-arch** [comment]
Can I merge my rp :)

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy live here: https://snapshot.org/#/strategy/sunder

**Pull request merged**